_stbt/core.py

```diff
@@ -1962,7 +1962,7 @@ def _get_sample(self, timeout_secs=10):
         return gst_sample
```

Comment:
Looks like you missed this spot -- you should be returning the result of `array_from_sample` from the previous line?